### PR TITLE
[Docs] Correct sidebarTitle in elasticsearch.mdx 

### PR DESCRIPTION
**Pull request by dscardona**
## Description

Correct sidebarTitle in elasticsearch.mdx

**Fixes** #5798

## Type of change

(Please delete options that are not relevant)

### What is the solution?

(Describe at a high level how the feature was implemented)
Changed sidebarTitle parameter from "Elastic" to "Elasticsearch"

### CODE REVIEW

File: docs/data-integrations/elasticsearch.mdx [hunk 1]
```diff
@@ -13,10 +13,10 @@ This handler is implemented using the `elasticsearch` library, the Python Elasti
 
 The required arguments to establish a connection are as follows:
 
-* `hosts` is the host name(s) or IP address(es) of the Elasticsearch server(s). If multiple host name(s) or IP address(es) exist, they should be separated by commas. This parameter is optional, but it should be provided if `cloud_id` is not.
-* `cloud_id` is the unique ID to your hosted Elasticsearch cluster on Elasticsearch Service. This parameter is optional, but it should be provided if `hosts` is not.
-* `username` is the username used to authenticate with the Elasticsearch server. This parameter is optional.
-* `password` is the password used to authenticate the user with the Elasticsearch server. This parameter is optional.
+- `hosts` is the host name(s) or IP address(es) of the Elasticsearch server(s). If multiple host name(s) or IP address(es) exist, they should be separated by commas. This parameter is optional, but it should be provided if `cloud_id` is not.
```
Comment (martyna-mindsdb) [inline docs/data-integrations/elasticsearch.mdx:16]: You replaced `*` with `-`.
Please change it back to `*`. Thanks.

**Comment by martyna-mindsdb** [comment]
@dscardona 
Please see one review comment above.

Also, please sign the CLA here.

File: docs/data-integrations/elasticsearch.mdx (hunk 1 above)
Comment (dscardona) [reply docs/data-integrations/elasticsearch.mdx]: Sorry about that! I had "format on save" turned on in VS Code. I've reverted the dashes to asterisks.

**Comment by dscardona** [comment]
Got it! 

I have read the CLA Document and I hereby sign the CLA.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by martyna-mindsdb** [comment]
@dscardona 
Could you please leave a comment that contains only this line:

I have read the CLA Document and I hereby sign the CLA

**Pull request merged**